packages/button/src/Button/Button.tsx

```diff
@@ -119,8 +133,8 @@ const Root = <T extends TgphElement>({
         rounded={roundedMap[size]}
         data-tgph-button
         data-tgph-button-layout={layout}
-        data-tgph-button-active={active}
-        disabled={disabled}
+        data-tgph-button-state={state}
+        disabled={state === "disabled" || state === "loading"}
```

Comment:
This prop now has no styling side effects. Now all it does is functionally disables the button.